test/DynamoCoreWpfTests/DynamoCoreWpfTests.csproj

```diff
@@ -45,9 +45,8 @@
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Greg, Version=1.2.7240.28357, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\src\packages\Greg.1.2.7240.28357\lib\net47\Greg.dll</HintPath>
-      <Private>True</Private>
```

Comment:
@zeusongit - these changes from private = true to nothing default to false -when this flag is not set to anything it is like setting CopyLocal = true - which means the dll gets copied to the bin directory, I think in most cases we only want this dll to be copied to the bin folder - not any sub folders, so it looks to me like we have let DynamoPackages.csproj copy this file and set all other references to copyLocal False - ie Private = true...

long story short - can you set the other references to CopyLocal = false and leave DynamoPackages reference to Greg to CopyLocal = true?

Reply:
Ok updated.